src/commands/blobs/blobs-get.mjs

```diff
@@ -0,0 +1,64 @@
+// @ts-check
+import { promises as fs } from 'fs'
+import { resolve } from 'path'
+
+import { getStore } from '@netlify/blobs'
+
+import { chalk, error as printError } from '../../utils/command-helpers.mjs'
+import requiresSiteInfo from '../../utils/hooks/requires-site-info.mjs'
+
+/**
+ * The blobs:get command
```

Comment:
[question] are we intentionally not providing CLI equivalents of `getMetadata` and `getWithMetadata`?  If yes, why?

Reply:
Yes. Just reducing the scope of the first version.